packages/reporter/src/commands/command.tsx

```diff
@@ -343,7 +357,7 @@ class Command extends Component<Props> {
 
     return (
       <ul className='command-child-container'>
-        {_.map(model.children, (child) => (
+        {model.children.map((child) => (
```

Comment:
Is children guaranteed to be populated on model? I like the change to a standard map, but it was safe against undefined values before, just wanted to confirm.

Reply:
It's not super obvious, but we are checking it here. I can update this to make it cleaner: https://github.com/cypress-io/cypress/pull/20465/files/03b75937f73d056c5796aba5bdd504f1e09d8932#diff-fb0aaf9c972fca4aa414f95ac88ab9ad1619459edeab226100daaba5e55645d0R350